tests/unit/test_execution_environment_utils.py

```diff
@@ -0,0 +1,292 @@
+# (c) 2012-2019, Ansible by Red Hat
+#
+# This file is part of Ansible Galaxy
+#
+# Ansible Galaxy is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License as published by
+# the Apache Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Ansible Galaxy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License
+# along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
+
+import logging
+import os
+import pytest
+
+from galaxy_importer.utils import execution_environment as ee_utils
+
+
+logger = logging.getLogger(__name__)
+
+
+EE_YAML = """---
+version: 1
+dependencies:
+  galaxy: requirements.yml
+  python: requirements.txt
+  system: bindep.txt
+
+additional_build_steps:
```

Comment:
is the `additional_build_steps` expected to be included in the `docs_blob` with the contents of the files in `dependencies`, or is that not needed?

Reply:
Yes it's expected to be included.

Reply:
ah yes i see now